### DISCUSSION TITLE
IKEA E1743 pairs as unsupported device

### DISCUSSION THREAD

**Discussion by lcx** [Koenkk/zigbee2mqtt#10118] [Devices]
Hi, I tried several times to pair an IKEA E1743 but have no luck. Or actually it does pair but appears as unsupported device. 
I can even press buttons and trigger error messages that a button was pressed on the unsuported device. 
Am I doing something wrong? Ikea lights work flawless, hue remotes as well and even the Iluminize switch works great. 
I'm running Zigbee2MQTT v1.22.1 via home assistant.

```
Zigbee2MQTT:debug 2021-12-09 20:40:38: Received Zigbee message from '0x8cf681fffe506225', type 'commandOn', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
Zigbee2MQTT:warn  2021-12-09 20:40:38: Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'
Zigbee2MQTT:warn  2021-12-09 20:40:38: Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
```

<img width="841" alt="CleanShot 2021-12-09 at 20 50 03@2x" src="https://user-images.githubusercontent.com/891345/145465844-f9a07a02-f22e-407a-b7b5-e6df9a946d1b.png">

**Comment by Nobody931** [comment]
Hey, I have a related issue with these buttons (see #9117 comment#21 where I provide a log file and a Zigbee sniff-log), but I tried it again and wanted to provide a log file here as well, maybe it can help!
[E1743_log.txt](https://github.com/Koenkk/zigbee2mqtt/files/7693138/E1743_log.txt)
Regarding the log, sometime after the 4th pairing attempt, I entered the pairing state with the button again, hoping it would then finish the interview. In between I was pressing the button a few times, so that it may not enter sleep (I was reading that this could be an issue).
Hope this may help, cheers

**Comment by fribse** [comment]
I've been working on the same problem for a day now. Out of desperation I renamed the device to give it a name I recognized. And then it suddenly paired up???
